### PR TITLE
Duplicate document detection

### PR DESCRIPTION
**Pull request by andrewconner**
For now, just generates a simple report listing the similar URIs with scores.

### CODE REVIEW

File: shoebox/app/com/keepit/scraper/DuplicateDocumentDetection.scala [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package com.keepit.scraper
+
+import com.keepit.model.{NormalizedURI, ScrapeInfo}
+import com.keepit.common.db.CX
+import play.api.Play.current
+import javax.xml.bind.DatatypeConverter._
```
Comment (eishay) [inline shoebox/app/com/keepit/scraper/DuplicateDocumentDetection.scala:5]: why do we need this import?

Comment (andrewconner) [inline shoebox/app/com/keepit/scraper/DuplicateDocumentDetection.scala:5]: For `parseBase64Binary`

**Comment by eishay** [comment]
do you have a test for it?

File: shoebox/app/com/keepit/scraper/DuplicateDocumentDetection.scala (hunk 1 above)
Comment (eishay) [inline shoebox/app/com/keepit/scraper/DuplicateDocumentDetection.scala:5]: oh, the javax.xml part alarmed me :-)

**Pull request merged**